### PR TITLE
Rspec test

### PR DESCRIPTION
**Pull request by jdmartinez1062**
Rspec tests for the winner methods in the Tic Tac Toe game. #winner and #somebody_win?

### CODE REVIEW

File: spec/test_spec.rb [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+require './lib/board.rb'
+require './lib/game.rb'
+require './lib/player.rb'
+
+describe 'Tic-tac-toe' do
```
Comment (bolah2009) [inline spec/test_spec.rb:7]: Let's add a test for `#next_player`?

Commenting out the code below passes the test.

https://github.com/jdmartinez1062/Tic-Tac-Toe/blob/73c1f4b6516d5241aabf732ed3b1feae2fc4d603/lib/game.rb#L24-L30

File: spec/test_spec.rb [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      draw[:b1] = '-'
+      expect(board.somebody_win?).to eql('x')
+    end
+    it 'tells you if nobody has won' do
+      draw1
+      expect(board.somebody_win?).to eql(false)
+    end
+  end
+
+  describe '#winner' do
```
Comment (bolah2009) [inline spec/test_spec.rb]: The method tested in this `describe` block is `winner_mock` not `winner`. 

Could we rename this to `winner_mock` and create a test for `winner`? :thinking:

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Hi Team :wave: 

Awesome job with the test. :clap: 

Let's add more test cases to handle edge cases, see comments below for examples.

#### A reminder from [pathwright Project specific](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/):

> Wrap specs for each method in `describe` block, hints.
> Use one `it` block for each expected behaviour. Do not test multiple aspects in one `it` block.
> Use your common sense: test positive and negative scenarios.


#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

File: spec/test_spec.rb (hunk 2 above)
Comment (jdmartinez1062) [reply spec/test_spec.rb]: We created winner_mock to test winner method functionality with out having to use a block, which only funtion is to print out the results. Thats why winner mock actually have 'puts' even though is not a method on main.rb, which according to previous project requirements wasn't allowed.

**Review by uche-inyama — APPROVED** [review]
## Hi Martinez!

Go ahead and merge with development branch, this project has been approved.

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```
